docs/infra/website.md

```diff
@@ -215,7 +237,7 @@ Some projects use custom HTML (in-line or entirely) to have a better show effect
 If using custom HTML, rememberto set the `excerpt` Front Matter.
```

Comment:
Could you please space between remember and to.

Reply:
@10eMyrT Good catch. Fixed in commit 5448fdc0b2875961f3215ec2c2169b8445c44356.